microk8s-resources/actions/common/utils.sh

```diff
@@ -1,5 +1,13 @@
 #!/usr/bin/env bash
 
+exit_if_no_sudo() {
+  # test if we can access the default kubeconfig
+  if ! cat $SNAP_DATA/credentials/client.config 2>&1 > /dev/null; then
```

Comment:
This would be better as:
    if [ ! -r $SNAP_DATA/credentials/client.config ]; then

Reply:
Thanks